solidity/contracts/NetworkSettings.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: SEE LICENSE IN LICENSE
+pragma solidity 0.6.12;
+
+import "./INetworkSettings.sol";
+import "./utility/Owned.sol";
+import "./utility/Utils.sol";
+
+/**
+ * @dev This contract maintains the network settings.
+ */
+contract NetworkSettings is INetworkSettings, Owned, Utils {
+    uint32 private constant PPM_RESOLUTION = 1000000;
+
+    address private _feeWallet;
+    uint32 private _feePortion;
+
+    // ensures that the portion is valid
+    modifier validPortion(uint32 _portion) {
+        _validPortion(_portion);
+        _;
+    }
+
+    // error message binary size optimization
+    function _validPortion(uint32 _portion) internal pure {
```

Comment:
We use these modifiers in 3 places already, so perhaps it should go to Utils?

Reply:
Yeah, same goes for `uint32 internal constant PPM_RESOLUTION = 1000000;` IMO.

Reply:
👍

Reply:
Done.

Reply:
Can you also remove it from other contracts which reference it?

Reply:
Reference what?

Reply:
Mi scusi!